src/main.rs

```diff
@@ -47,5 +66,10 @@ fn run() -> Result<(), failure::Error> {
 
     let args = Cli::from_args();
     run_wasm_pack(args.cmd)?;
+
+    if let Ok(update_available) = update_available.try_recv() {
+        println!("There's a newer version of wasm-pack available, the new version is: {}, you are using: {}", update_available.1, update_available.0);
```

Comment:
Maybe link to the releases page and/or suggest cargo install wasm-pack -f

Reply:
Yes, let's definitely link to https://rustwasm.github.io/wasm-pack/installer/ and maybe even inline the install command directly.

@ashleygwilliams @alexcrichton How stable do we expect the install command to be? Should we just print it directly right here?

Reply:
I think for now somewhat unstable enough that we should probably direct to the website